.gitignore

```diff
@@ -1,7 +1,8 @@
-/.output
-/PSModules
-/.vscode
+/.output/
+/PSModules/

```

Comment:
I'm starting to think Whiskey should be downloading its build dependencies to an AppData directory instead of the local build directory. Would remove the need to always `.gitignore` this directory and have the benefit that Whiskey only needs to download a dependency once per machine instead of once per project.